types/supercluster/index.d.ts

```diff
@@ -33,6 +33,20 @@ export interface Options {
      * Whether timing info should be logged.
      */
     log?: boolean;
+    /**
+     * initial determines the starting values for the accumulator for the reducer.
```

Comment:
Why not use the same wording as the source code ([`supercluster/index.js`](https://github.com/mapbox/supercluster/blob/master/index.js#L26-L33))

```ts
/**
 * a reduce function for calculating custom cluster properties
 *
 * @example
 * function (accumulated, props) { accumulated.sum += props.sum; }
 */
reduce?: (accumulated: any, props: any) => void;

/**
 * initial properties of a cluster (before running the reducer)
 *
 * @example
 * function () { return {sum: 0}; }
 */
initial?: () => any;

/**
 * properties to use for individual points when running the reducer
 *
 * @example
 * function (props) { return {sum: props.my_value}; }
 */
map?: (props: any) => any;
```

Reply:
will change to the comment from the source code thanks!